src/main/java/com/jagrosh/giveawaybot/entities/Giveaway.java

```diff
@@ -67,7 +69,7 @@ public Giveaway(long messageId, long channelId, long guildId, long userId, Insta
         this.expanded = expanded;
     }
     
-    public Message render(Color color, Instant now)
+    public Message render(Color color, String emoji, Instant now)
```

Comment:
This shouldn't be passed in here, this should be a field on the giveaway itself and should also be stored in every giveaway in the database (so that the emoji can be swapped even if some giveaways are already running). Make sure that it stores null if the default emoji is being used to save some space.